pkg/storage-init/storage-init.sh

```diff
@@ -302,7 +293,12 @@ for BLK_DEVICE in $BLK_DEVICES; do
 done
 
 #Recording SMART details to a file
-SMART_JSON=$(smartctl -a "$(grep -m 1 /persist < /proc/mounts | cut -d ' ' -f 1)" --json)
+if [ -L /dev/root ] ; then
```

Comment:
Are there cases when we wouldn't have a /dev/root?
If so could that change from one reboot to the next? (That would confuse the PREVIOUS_FILE comparisons on power cycle counter).

Reply:
OK